spring-integration-feed/src/test/java/org/springframework/integration/feed/inbound/FeedEntryMessageSourceTests.java

```diff
@@ -88,6 +88,40 @@ public void testReceiveFeedWithEntriesSorted() throws Exception {
 		assertTrue(time2 < time3);
 		assertNull(source.receive());
 	}
+	
+	// verifies that when entry has been updated since publish, that is taken into 
+	// account when determining if the feed entry has been seen before
+	@Test
+	public void testReceiveAtomFeedWithEntryHavingBeenUpdatedAfterPublishAndRepeatWithPersistentMetadataStore() throws Exception {
+		URL url = new URL("file:src/test/java/org/springframework/integration/feed/atom.xml");
```

Comment:
H-m. It's dangerous.
Let's change it to the CLASSPATH:
````java
URL url = new ClassPathResource("org/springframework/integration/feed/atom.xml").getURL();
````

And if you don't mind, please, fix all other tests for the same issue.